### PR TITLE
Add stability test for infinite looping

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

File: perf/stability/looper/main.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	"sync"
+)
+
+var (
+	wg = sync.WaitGroup{}
+)
+
+func main() {
+	// Simple server that listens on many ports
+	wg.Add(4)
```
Comment (mandarjog) [inline perf/stability/looper/main.go]: why 4? not 6?

File: perf/stability/looper/templates/client.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            }
+            POD_IP=""
+            while [[ -z $POD_IP ]]; do
+              echo "attempting to get POD_IP"
+              POD_IP=$(kubectl get pods -l app=server -n istio-stability-looper -ojsonpath='{.items[0].status.podIP}')
+              sleep 1
+            done
+            while true; do
+              echo "`date`: Sending traffic"
+              assert 200 $(curl -s -o /dev/null -w "%{http_code}\n" "${POD_IP}:8085")
```
Comment (mandarjog) [inline perf/stability/looper/templates/client.yaml]: 1. can you also add case for `-H "Host: server"`
2. Can you add a note on   *5 is service + containerPort,   *6 containerOnly,  and *7 is undeclared ports.

File: perf/stability/looper/main.go (hunk 1 above)
Comment (howardjohn) [reply perf/stability/looper/main.go]: oops, I added one later

File: perf/stability/looper/templates/client.yaml (hunk 2 above)
Comment (howardjohn) [reply perf/stability/looper/templates/client.yaml]: yeah, I added them now

**Pull request merged**